apps/designer/app/routes/rest/pages.$projectId.ts

```diff
@@ -113,3 +172,12 @@ export const loader: LoaderFunction = async ({
     };
   }
 };
+
+const fieldError = (fieldName: string, error: string) => {
```

Comment:
we should normalize this kind of utilities in a shared thing, because its not a singular use case, this should be useful in a lot of places, we need form-utils